CatAnalyzer/src/BTagWeightEvaluator.cc

```diff
@@ -118,7 +124,8 @@ double BTagWeightEvaluator::getSF(const cat::Jet& jet, const int unc) const
   const double aeta = std::abs(eta);
   if ( pt <= 20 or aeta >= 2.4 ) return 1.0;
 
-  double discr = jet.bDiscriminator(btagAlgo_);
+  double discr = 0;
+  for ( unsigned int i; i < btagAlgo_.size(); i++) discr += jet.bDiscriminator( btagAlgo_.at(i) );
```

Comment:
sum over discr... is this well defined?

Reply:
https://twiki.cern.ch/twiki/bin/viewauth/CMS/BtagRecommendation102X
Should use sum of discriminators